recipe/meta.yaml

```diff
@@ -23,17 +25,14 @@ requirements:
     - zstandard
     - python-xxhash
     # Other common requirements
-    - xrootd  # [not win]
```

Comment:
I thought the idea around removing this was so it could be noarch?

Reply:
noarch will cause it to only test on linux. To the best of my knowledge, there's still no workaround for adding other platforms for testing.